src/palace/manager/celery/tasks/axis.py

```diff
@@ -49,6 +49,13 @@ def import_all_collections(
             )
             list_identifiers_for_import.apply_async(
                 kwargs={"collection_id": collection.id},
+                eta=utc_now()
+                + timedelta(
+                    seconds=count * 5
+                ),  # stagger the execution of the collection import
+                # in order to minimize chance of deadlocks caused by
+                # simultaneous updates to the metadata when CM has multiple
+                # axis collections configured with overlapping content
```

Comment:
Minor: Instead of having to work with a `timedelta` here, you could just use the [`countdown` parameter](https://docs.celeryq.dev/en/stable/userguide/calling.html#eta-and-countdown).
```suggestion
                delay=count * 5,  # stagger the execution of the collection import
                # in order to minimize chance of deadlocks caused by
                # simultaneous updates to the metadata when CM has multiple
                # axis collections configured with overlapping content
```

Reply:
I'll give it a shot - but it looks like we should be using `countdown` not `delay` unless I'm missing something.